docs/log_format.md

```diff
@@ -14,32 +14,32 @@ Shadow produces simulator log messages in the following format:
 real-time [thread-id:thread-name] virtual-time [loglevel] [hostname:ip] [src-file:line-number] [function-name] MESSAGE
 ```
 
-- `real-time`:  
+- `real-time`:
```

Comment:
I think these trailing whitespaces are being used to force a newline in the output. I'm fine with dropping that newline, but wasn't sure whether that was your intent :)

Reply:
Yeah, I noticed that too. I think at some point I set my VS Code to cut trailing whitespace when saving a file, which is why that happened. I was going to undo it, and then on second thought decided it didn't matter.